site/bulletin_board/views/user_view.ejs

```diff
@@ -0,0 +1,101 @@
+                            <a>join at</a>
+                            </div>
+                            <div>
+                            <a style="font-size: 10px; color: #637987;"><%= user.join_data %></a>
+                            </div>
+                        </div>
+                    </div>
+                <div class="col-6">
+                    <div class="profile_left_side_text">
+                        <div>
+                        <a>place</a>
+                        </div>
+                        <div>
+                        <a style="font-size: 10px; color: #637987">iraq</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="profile_left_side_text_emojy" style="padding-top: 10%;">
+                <i class='far fa-id-badge' style='font-size:20px'>
+                <span><%if (user.firstname){%>
+                    <%= user.firstname %>
+                    <%}else{%>
+                    <%= "privet" %>
+                    <% } %>
+
+                </span></i>
+            </div>
+            <div class="profile_left_side_text_emojy">
+                <i class='far fa-calendar-alt' style='font-size:20px'></i>
+                <span style=" font-size:20px ">
+                    <%if (user.birthdate){%>
+                        <%= user.birthdate %>
+                        <%}else{%>
+                        <%= "privet" %>
+                        <% } %>
+                </span>
+            </div>
+            <div class="profile_left_side_text_emojy">
+                <i class='far fa-heart' style='font-size:20px'></i>
+                <span  style=" font-size:20px ">
+                    <%if (user.gender){%>
+                        <%= user.gender %>
+                        <%}else{%>
+                        <%= "privet" %>
+                        <% } %>                </span>
+            </div>
+            <div>
+                <img src="https://i.ibb.co/McQjdQf/Group-1.png" alt="Group-1" class="profile_enter_image">    
+                </div>
+        </div>
+
+        <div class="col-12 col-md-9" style="background-color: #202D36;">
+            <div class="container py-5">
+                <div style="color: white;"><h2>Last Post</h2></div>
+                <div class="custom-scrollbar">
+                    <% posts.forEach(function(post){ %>
+                        <%- include('view_profile_poster', { post: post }); %>
+                      <% }); %>
+        </div>
+    </div>
+    <div style="color: white;"><h2>Post you like</h2></div>
+    <div class="custom-scrollbar">
+        <% posts_liked.forEach(function(post){ %>
+            <%- include('view_profile_poster', { post: post }); %>
+          <% }); %>
+
+</div>
+      </div>      
+</div>
+<%- include('footer', { javascript_uri: '/javascripts/create_post.js' }); %>
+
+<%- include('footer', { javascript_uri: '/javascripts/view_post.js' }); %>
```

Comment:
It looks like you're including the footer here twice. I think if you view this page's source in your browser you will see two blocks of footer markup at the bottom of your page which would make your HTML malformed. 

Perhaps instead of:

```
<%- include('footer', { javascript_uri: '/javascripts/create_post.js' }); %>
<%- include('footer', { javascript_uri: '/javascripts/view_post.js' }); %>
```

you might consider something like:

```
<%- include('footer', { javascript_uris: [ '/javascripts/create_post.js', '/javascripts/view_post.js' ] }); %>
```

Which would require a small change to the `footer` template, but would solve the problem of how to include multiple `script` tags at the end of the document.

By the way, I think putting the script down here was a bit of a hack on my part. Probably best practice is to use [this](https://api.jquery.com/ready/) 🤷  ... I think I wanted an excuse to have a template parameter. Following the pattern here on this site makes sense because it's already working that way - but I just wanted to point out my hack to avoid any misunderstanding about this being a good assumption (putting scripts down here and then assuming the document is ready because they are loaded is not a very robust  assumption).